app/models/driver.rb

```diff
@@ -0,0 +1,33 @@
+class Driver < ApplicationRecord
+  has_many :trips
+
+  validates :name, presence: { message: "Name cannot be blank!"}
+  validates :vin, presence: { message: "VIN cannot be blank!"}
+
+  paginates_per 10
+
+  def total_money
+    total = super.to_f
+    total = total - (total * 0.15)
+    return '%.2f' % total
+  end
+
+  def average_rating
+    if trips.any? == false
+      return "Nil"
```

Comment:
It would be better to return either `nil` itself or a string describing the type of issue rather than the string of "Nil" which is not meaningful to your users